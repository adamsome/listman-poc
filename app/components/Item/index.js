import React from 'react';

const Item = (props) => {
  return (
    <div>
      <p>{ props.title }</p>
    </div>
  )
}

export default Item
