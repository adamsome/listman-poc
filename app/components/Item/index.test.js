import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Item from './index'

let wrapper
beforeEach(() => {
  wrapper = shallow(<Item title='One' />)
})

it('renders correctly', () => {
  expect(toJson(wrapper)).toMatchSnapshot()
})

it('shows title', () => {
  expect(wrapper.type()).toEqual('div')

  let title = wrapper.childAt(0)
  expect(title.type()).toEqual('p')
  expect(title.text()).toEqual('One')
})

