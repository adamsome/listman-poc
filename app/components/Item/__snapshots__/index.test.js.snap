exports[`test renders correctly 1`] = `
<div>
  <p>
    One
  </p>
</div>
`;
